src/components/Tag/Tag.styles.ts

```diff
@@ -22,6 +22,7 @@ import {
     inactiveStatus,
     initialStyles,
     regular,
+    boarder,
```

Comment:
please add props in alphabetical order, there is a plugin you can use. Check the contribute file please =]

Reply:
also i got some feedback from design, can you change 'boarder' to 'chips' please

Reply:
done